lib/registerPlugins.js

```diff
@@ -65,29 +65,15 @@ function registerResourcePlugins(server, config, callback) {
 }
 
 /**
- * Revise npm packages's scope name
- * @method reviseScope
- * @param  {String} scope           Name of npm package's scope
- * @return {String}                 Revised scope name
- */
-function reviseScope(scope) {
-    if (/^[@].*/.test(scope)) {
-        return scope;
-    }
-
-    return `@${scope}`;
-}
-
-/**
  * Require notification plugin
  * @method requireNotificationPlugin
  * @param  {Object}    config           Configuration object for notification plugins.
  * @param  {String}    plugin           Notification plugin name.
  * @return {Function}                   Function of notifier
  */
 function requireNotificationPlugin(config, plugin) {
-    if (config[plugin].scope) {
-        return require(`${reviseScope(config[plugin].scope)}/screwdriver-notifications-${plugin}`);
+    if (config[plugin].module) {
```

Comment:
I think the keyword `module` is kinda confusing here. This one is used only when you want to load some scoped package right? 
I'm not sure what's a better keyword. Maybe `scopedPackage`?

Reply:
I think this property is used only when we want to load scoped package.
Fixed to `scopedPackage`.